Thai/Travel/Travel.md

```diff
@@ -0,0 +1,7 @@
+﻿You will fall in love with Thailand. It is amazingly beautiful. We have plenty of stunning beaches and crystal clear water. Imagine you are at the sea now, what would you like to do?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.